src/main/java/org/sasanlabs/service/vulnerability/xss/reflected/XSSInImgTagAttributeInjectionVulnerability.java

```diff
@@ -169,17 +195,47 @@ public UrlParamBasedImgTagAttrInjection() {
             htmlTemplate = "LEVEL_1/XSS")
     public ResponseEntity<String> getVulnerablePayloadLevelSecure(
             @RequestParam Map<String, String> queryParams) {
+
         String vulnerablePayloadWithPlaceHolder = "<img src=\"%s\" width=\"400\" height=\"300\"/>";
         StringBuilder payload = new StringBuilder();
 
         for (Map.Entry<String, String> map : queryParams.entrySet()) {
-            if (allowedValues.contains(map.getValue())) {
+
+            if (isSecure(map.getValue())) {
```

Comment:
This is not the right way to protect against XSS in an image src attribute.

Level5 and Level7 are the secure implementations of protection against XSS in image src attribute.
So can you please make Level5 as Level6 and Level6 as Level5 and mark it as Secure.
e.g
```
    @VulnerableAppRequestMapping(
            value = LevelConstants.LEVEL_5,
            **variant = Variant.SECURE,**
            htmlTemplate = "LEVEL_1/XSS")
```